cmd/minishift/cmd/oc_env.go

```diff
@@ -42,12 +45,19 @@ type OcShellConfig struct {
 }
 
 func getOcShellConfig(ocPath, forcedShell string) (*OcShellConfig, error) {
+	var cmdLine string
+
 	userShell, err := shell.GetShell(forcedShell)
 	if err != nil {
 		return nil, err
 	}
 
-	cmdLine := "minishift oc-env"
+	if cmdProfile.ProfileExists() {
+		cmdLine = "minishift oc-env --profile " + cmdProfile.GetProfileName()
```

Comment:
I don't get what's going here? Did we not say that profile is a global option? Also if the profile is set/activated I would expect that I can just run `minishift oc-env`, right? Only if I explicitly run `minishift --profile foo oc-env`, I would expect the need to pass through the profile, but in this case it should become `minishift --profile foo oc-env`.